browser/ui/brave_shields_data_controller.cc

```diff
@@ -49,14 +59,64 @@ int BraveShieldsDataController::GetTotalBlockedCount() {
           resource_list_blocked_fingerprints_.size());
 }
 
-bool BraveShieldsDataController::GetIsBraveShieldsEnabled() {
+bool BraveShieldsDataController::IsBraveShieldsEnabled() {
   auto* map = HostContentSettingsMapFactory::GetForProfile(
       web_contents()->GetBrowserContext());
   return brave_shields::GetBraveShieldsEnabled(map, GetCurrentSiteURL());
 }
 
 GURL BraveShieldsDataController::GetCurrentSiteURL() {
-  return web_contents()->GetURL();
+  return web_contents()->GetLastCommittedURL();
+}
+
+AdBlockMode BraveShieldsDataController::GetAdBlockMode() {
```

Comment:
please add tests (unit preferably) for these methods

Reply:
added